src/test/java/com/wikia/webdriver/testcases/signuptests/SignUpTests.java

```diff
@@ -98,34 +98,34 @@ public void anonCanSignUp() {
     preferences.verifyEmailMeSection();
   }
 
-  @Test(groups = {"Signup_anonCanSignUpWhenCreatingNewWiki", "SignUp"})
+  @Test(groups = {"Signup_anonCanSignUpWhenCreatingNewWiki", "SignUp"}, enabled = false)
   @RelatedIssue(issueID = "SOC-2283")
   public void anonCanSignUpWhenCreatingNewWiki() {
     CreateNewWikiPageObjectStep1 createNewWiki1 = new CreateNewWikiPageObjectStep1(driver).open();
     createNewWiki1.disableCaptcha();
     String wikiName = createNewWiki1.getWikiName();
     createNewWiki1.typeInWikiName(wikiName);
     createNewWiki1.verifySuccessIcon();
-    CreateNewWikiLogInSignUpPageObject cnwSignUpPage = createNewWiki1.submitToLogInSignUp();
-    SignUpPageObject signUp = cnwSignUpPage.submitSignup();
-    String userName = "User" + signUp.getTimeStamp();
-    String password = "Pass" + signUp.getTimeStamp();
-    String email = credentials.emailQaart2;
-    String emailPassword = credentials.emailPasswordQaart2;
-
-    signUp.typeEmail(email);
-    signUp.typeUserName(userName);
-    signUp.typePassword(password);
-    signUp.enterBirthDate(PageContent.WIKI_SIGN_UP_BIRTHMONTH, PageContent.WIKI_SIGN_UP_BIRTHDAY,
-        PageContent.WIKI_SIGN_UP_BIRTHYEAR);
-    AlmostTherePageObject almostTherePage = signUp.submit(email, emailPassword);
-    almostTherePage.verifyAlmostTherePage();
-    ConfirmationPageObject confirmPageAlmostThere =
-        almostTherePage.enterActivationLink(email, emailPassword, wikiCorporateURL);
-    confirmPageAlmostThere.typeInUserName(userName);
-    confirmPageAlmostThere.typeInPassword(password);
-    createNewWiki1 = confirmPageAlmostThere.CNWSubmitButton(email, emailPassword);
-    createNewWiki1.verifyWikiName(wikiName);
+//    CreateNewWikiLogInSignUpPageObject cnwSignUpPage = createNewWiki1.clickNextToSignIn();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3ACommentedOutCodeLine)